### ISSUE TITLE
Regarding related work on trying AL on fixed self-supervised model

### ISSUE THREAD

**Issue by sieu-n** [zeyademam/active_learning#2]
In the end of section 3 of your paper, you described
"Consequently,  it  is  important  to  evaluate  AL  algorithms when the feature extractor is fixed and only the classifica-tion head is finetuned. This task is a common benchmark inself-supervised learning (SSL) research [13, 14, 26], wherethe proposed SSL method is used to pretrain the network’sfeature extractor, then a linear classification head is trained on the features in a fully-supervised fashion.  However, tothe best of our knowledge, AL strategies on ImageNet havenot been evaluated in this specific setting."

There actually seems to be a previous work that did just this. 
Please check out the paper "Reducing Label Effort: Self-Supervised meets Active Learning" published on August, 2021.
link: https://arxiv.org/abs/2108.11458
![image](https://user-images.githubusercontent.com/48239275/144192772-0ec9e699-2867-4149-b704-d719c82331c5.png)

**Comment by zeyademam** [comment]
Thank you for pointing out this concurrent work! We will read it more carefully and consider discussing it in our next revision. Also, we want to point out that this paper does not consider ImageNet, it only addresses tiny ImageNet. 

Combining SSL and active learning is by no means new, and our work simply proposes an approach which is scalable to large-scale datasets, such as ImageNet.
